tests/Orm/AssociationIpsumSortTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace EasyCorp\Bundle\EasyAdminBundle\Tests\Orm;
+
+use EasyCorp\Bundle\EasyAdminBundle\Tests\TestApplication\Controller\AssociationIpsumCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Tests\TestApplication\Controller\DashboardController;
+use EasyCorp\Bundle\EasyAdminBundle\Tests\TestApplication\Entity\AssociationIpsum;
+use Symfony\Component\DomCrawler\Crawler;
+
+class AssociationIpsumSortTest extends AbstractAssociationTest
+{
+    private $repository;
+
+    protected function getControllerFqcn(): string
+    {
+        return AssociationIpsumCrudController::class;
+    }
+
+    protected function getDashboardFqcn(): string
+    {
+        return DashboardController::class;
+    }
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->client->followRedirects();
+        $this->repository = $this->entityManager->getRepository(AssociationIpsum::class);
+    }
+
+    /**
+     * @dataProvider sorting
+     */
+    public function testSorting(array $query, callable $sortFunction, string $expectedSortIcon)
```

Comment:
```suggestion
    public function testSorting(array $query, ?callable $sortFunction, string $expectedSortIcon)
```
We don't need to pass a closure. Use `null`, `asort`, and `arsort` respectively for each data. Then replace
```php
$expectedAmountMapping = $sortFunction($expectedAmountMapping);
```
with
```php
if (null !== $sortFunction) {
    $sortFunction($expectedAmountMapping);
}
```